templates/github/.ci/ansible/Containerfile.j2.copy

```diff
@@ -10,7 +10,7 @@ ADD {{ item.source }} {{ item.source }}
 # Install python packages
 # Hacking botocore (https://github.com/boto/botocore/pull/1990)
 
-RUN pip3 install \
+RUN pip3 install --no-binary=:all: \
```

Comment:
prefer tar.gz

Reply:
actually it breaks, because it can't install cryptography:
https://github.com/pulp/plugin_template/runs/7976724965?check_suite_focus=true#step:8:905

Reply:
let me try it